src/Sarif/HashUtilities.cs

```diff
@@ -7,11 +7,35 @@
 
 namespace Microsoft.CodeAnalysis.Sarif
 {
+    public class HashData
+    {
+        string md5;
+        string sha1;
+        string sha256;
+
+        public HashData()
+        {
+            this.md5 = String.Empty;
```

Comment:
Conventions:

* We don't use "`this.`" because the naming convention tells us it's a field.
* We use the "type alias" rather than the class name for the built-in types ("`string`" rather than "`String`").

So:


```
    _md5 = string.Empty;